index.html

```diff
@@ -0,0 +1,127 @@
+      </p>
+      <p id="when">
+        2022.11.09(WED) ~ 12(SAT)
+      </p>
+      <p id="where">
+        Salle Evala, Hotel 2 Fevrier, Lomé - TOGO
+      </p>
+    </section>
+    <section id="main-program">
+      <h1>Main Program</h1>
+      <span class="program-bar"></span>
+      <ul class="program">
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Lecture</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Workshops</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Demo</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Conference</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Livestream</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+      </ul>
+      <button id="join-program">Join Challenge</button>
+      <a id="see-more-program" href="#contact-form">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="speakers">
+    </section>
+    <section id="partner">
+      <h1>Partner</h1>
+      <span class="partner-bar"></span>
+      <ul>
+        <li><img src="assets/images/sonsors/iamport.png" alt=""></li>
+        <li><img src="assets/images/sonsors/kakao.png" alt=""></li>
+        <li><img src="assets/images/sonsors/line-friends.png" alt=""></li>
+        <li><img src="assets/images/sonsors/mozilla.png" alt=""></li>
+      </ul>
+
+    </section>
+  </main>
+  <footer>
+    <img id="footer-img" src="assets/images/Lome_Fintech_LogoB_59.png" alt="">
+    <p>
+      @2022 Lome Fintech Challenge <br> Some Rights Reserved
+    </p>
+
+  </footer>
```

Comment:
- [ OPTIONAL ] I think here in the mobile version for the main page  you don't need to add the footer it is just found in the desktop version so kindly remove it from your project 💪🏻😊🙏🏻